lib/export_dsi_approvers_to_big_query.rb

```diff
@@ -22,7 +22,8 @@ def present_for_big_query(batch)
         email: user['email'],
         family_name: user['familyName'],
         given_name: user['givenName'],
-        school_urn: user.dig('organisation', 'urn')
+        school_urn: user.dig('organisation', 'urn'),
+        organisation_uid: user['uid']
```

Comment:
I'm fairly sure this will be in `user['organisation']['uid']`

Reply:
I thought so at first too, but it's user['uid'] for approvers

Reply:
That's not what I see here @josephhull676 :

```ruby
{"organisation"=>
    {"id"=>"FD0E5C93-FA9C-43F8-8967-000A36832AA0",
     "name"=>"Kingskerswell Church of England Primary School",
     "category"=>{"id"=>"001", "name"=>"Establishment"},
     "type"=>{"id"=>"03", "name"=>"Voluntary Controlled School"},
     "urn"=>"113398",
     "uid"=>nil,
     "ukprn"=>nil,
     "establishmentNumber"=>"3112",
     "status"=>{"id"=>1, "name"=>"Open"},
     "closedOn"=>nil,
     "address"=>"Coffinswell Lane, Kingskerswell, Newton Abbot, Devon, TQ12 5HN",
     "telephone"=>"01803873398",
     "region"=>{"id"=>"K", "name"=>"South West"},
     "phaseOfEducation"=>{"id"=>2, "name"=>"Primary"},
     "statutoryLowAge"=>4,
     "statutoryHighAge"=>11,
     "legacyId"=>"1019530",
     "companyRegistrationNumber"=>nil},
   "roleId"=>10000,
   "roleName"=>"Approver",
   "userId"=>"censored",
   "email"=>"censored@gmail.com",
   "familyName"=>"censored",
   "givenName"=>"censored"}
```

(censored some of the user's personal details)

Reply:
Yup, sorted now and pretty obvious now I look at it again.